hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -1,10 +1,11 @@
+share/ozone/lib/FastInfoset.jar
```

Comment:
Please try to avoid reordering the jar-report.  These unnecessary changes make it more difficult to see what has actually changed.

Reply:
I just ran the script and it automatically updated this. I can remove the file, if it's too cumbersome. But, as long as the CI passes, we know that the data in `jar-report.txt` is accurate, no?

Reply:
You don't need to run `hadoop-ozone/dist/src/main/license/update-jar-report.sh`.  Running `hadoop-ozone/dev-support/checks/dependency.sh` shows you the differences.  Just update `jar-report.txt` manually, keeping current order, inserting/removing items if needed.

Reply:
There still are unnecessarily reordered lines.  On closer look, no changes should be done in the jar report.